### ISSUE TITLE
bug: The ... relation (alias relation) is not implicitly defined 

### ISSUE THREAD

**Issue by gruuya** [authorizer-tech/access-controller#39]
**Is your feature request related to a problem? Please describe.**

The [SubjectSet rewrites example](https://www.authorizer.tech/docs/overview/examples/hierarchy) is not working since the `...` relation (alias relation) is not implicitly defined as [stated in the docs](https://www.authorizer.tech/docs/overview/concepts/relation-tuples#the--relation-alias-relation). Example:
```bash
$ grpcurl -plaintext -d \
'{
    "config": {
        "name": "projects",
        "relations": [
            {
                "name": "viewer"
            }
        ]
    }
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.NamespaceConfigService.WriteConfig
{

}
$ grpcurl -plaintext -d \
'{
    "config": {
        "name": "tasks",
        "relations": [
            {
                "name": "parent"
            },
            {
                "name": "viewer",
                "rewrite": {
                    "union": {
                        "children": [
                            {
                                "tupleToSubjectset": {
                                    "tupleset": { "relation": "parent" },
                                    "computedSubjectset": { "relation": "viewer" }
                                }
                            }
                        ]
                    }
                }
            }
        ]
    }
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.NamespaceConfigService.WriteConfig
{

}
$ grpcurl -plaintext -d \
'{
    "relationTupleDeltas": [
        {
            "action": "ACTION_INSERT",
            "relationTuple": {
                "namespace": "tasks",
                "object": "task1",
                "relation": "parent",
                "subject": {
                    "set": {
                        "namespace": "projects",
                        "object": "project1",
                        "relation": "..."
                    }
                }
            }
        }
    ]
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.WriteService.WriteRelationTuplesTxn
ERROR:
  Code: InvalidArgument
  Message: SubjectSet 'projects:project1#...' references relation '...' which is undefined in the namespace 'projects' at snapshot config timestamp '2021-08-30 12:55:24.160712 +0000 UTC'. If this relation was recently added to the config, please try again in a couple minutes
```

**Describe alternatives you've considered**

Once I add the `...` relation explicitly the rewrite example works:
```bash
$ grpcurl -plaintext -d \
'{
    "config": {
        "name": "projects",
        "relations": [
            {
                    "name": "viewer"
            },
            {
                    "name": "..."
            }
        ]
    }
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.NamespaceConfigService.WriteConfig
{

}
$ grpcurl -plaintext -d \
'{
    "relationTupleDeltas": [
        {
            "action": "ACTION_INSERT",
            "relationTuple": {
                "namespace": "tasks",
                "object": "task1",
                "relation": "parent",
                "subject": {
                    "set": {
                        "namespace": "projects",
                        "object": "project1",
                        "relation": "..."
                    }
                }
            }
        },
        {
            "action": "ACTION_INSERT",
            "relationTuple": {
                "namespace": "projects",
                "object": "project1",
                "relation": "viewer",
                "subject": {
                    "id": "gruuya"
                }
            }
        }
    ]
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.WriteService.WriteRelationTuplesTxn
{

}
$ grpcurl -plaintext -d \
'{
    "namespace": "tasks",
    "object": "task1",
    "relation": "viewer",
    "subject": {
        "id": "gruuya"
    }
 }' localhost:50052 authorizer.accesscontroller.v1alpha1.CheckService.Check
{
  "allowed": true
}
```

**Comment by jon-whit** [comment]
Good catch! Thanks for reporting this.

I'll get the fix in and a new patch release cut tonight 👍🏻

**Comment by jon-whit** [comment]
Fixed in [release `v0.1.5`](https://github.com/authorizer-tech/access-controller/releases/tag/v0.1.5).
